README.md

```diff
@@ -420,6 +420,18 @@ pct install https://packages.mycompany.com/pct/my-template-1.2.3.tar.gz
 
 Will attempt to download the PCT template from the specified url and then afterward install it like any other locally available PCT template archive.
 
+#### Remote Git Repository
+
+**Git** must be installed for this feature to work. The git repository must contain only one template and must be structured with the `pct-config.yml` file and the `content` directory in the root directory of the repository.
+
+For example, this command:
+
+```bash
+pct install --git-uri https://github.com/puppetlabs/pct-test-template-01.git
+```
+
+Will attempt to clone the PCT template from the git repository at the specified uri and install to the default template location.
```

Comment:
```suggestion
This will attempt to clone the PCT template from the git repository at the specified URI and install to the default template location.
```